### PR TITLE
Update forum ELB to use HTTP instead of TCP

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: stanford/roles/cluster/defaults/main.yml [hunk 1]
```diff
@@ -230,14 +230,14 @@ CLUSTER_ELBS:
   forum:
     - name: "{{ COMMON_DEPLOYMENT }}-elb-forum-private"
       health_check:
-        ping_protocol: tcp
-        ping_port: 80
```
Comment (caesar2164) [inline stanford/roles/cluster/defaults/main.yml]: don't we need the `ping_port`?

Comment (stvstnfrd) [reply stanford/roles/cluster/defaults/main.yml]: I think it's optional/implied due to `http`, but we do include it explicitly for the other ELBs, so I'll add it back.

**Comment by kluo** [comment]
+1

**Comment by caesar2164** [comment]
cool, 👍

**Pull request merged**